### ISSUE TITLE
Convert BML files under htdocs/support

### ISSUE THREAD

**Issue by afuna** [dreamwidth/dw-free#1046]

**Comment by hotlevel4** [comment]
Trying to convert support/faq.

**Comment by kareila** [comment]
Hi @hotlevel4, are you interested in converting any more of these?  If not, let me know so I can put this back up for grabs!

**Comment by kareila** [comment]
lol never mind :stuck_out_tongue_winking_eye:

**Comment by hotlevel4** [comment]
I am pretty sporadic in my work on this, though, so if it's easier for you to unassign and see if anybody else wants to pick it up and complete it in a more timely fashion, I'm happy to pop in and just add things if/when I have time.

**Comment by kareila** [comment]
There's no rush to get it done, though, so as long as you're interested in continuing to work on it, feel free to hang onto it!  There are plenty of unconverted pages to go around...

**Comment by rahaeli** [comment]
Yes, please never apologize for the speed at which you are moving! We just have a tendency for people to think "I don't really want to do this anymore, but I don't want to drop the issue because then I am LETTING PEOPLE DOWN" and we like to make sure people know that it's okay to throw things back in the pool. :)
